_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-instagram-gallery.php

```diff
@@ -45,25 +45,27 @@ public function register_routes() {
 			$this->namespace,
 			$this->rest_base . '/connect-url',
 			array(
-				'methods'  => WP_REST_Server::READABLE,
-				'callback' => array( $this, 'get_instagram_connect_url' ),
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_instagram_connect_url' ),
+				'permission_callback' => '__return_true',
 			)
 		);
 
 		register_rest_route(
 			$this->namespace,
 			$this->rest_base . '/connections',
 			array(
-				'methods'  => WP_REST_Server::READABLE,
-				'callback' => array( $this, 'get_instagram_connections' ),
+				'methods'             => WP_REST_Server::READABLE,
+				'callback'            => array( $this, 'get_instagram_connections' ),
+				'permission_callback' => '__return_true',
 			)
 		);
 
 		register_rest_route(
 			$this->namespace,
 			$this->rest_base . '/gallery',
 			array(
-				'args'     => array(
+				'args'                => array(
```

Comment:
this indentation looks a bit off, but I guess if phpcs think it's fine, then it's fine 🤷

Reply:
yeah, phpcs added that one. It's to match the other array keys after the args array that are at the same level below it.